GUIDELINES.md

```diff
@@ -0,0 +1,7 @@
+# :+1::tada: Thanks for Contributing! :tada::+1:
+
+See [openstax/CONTRIBUTING.md](https://github.com/openstax/napkin-notes/CONTRIBUTING.md) for more information on how to contribute to openstax!
```

Comment:
Can you name this file `CONTRIBUTING.md`? It then shows up in places like when you create an Issue or Pull Request.